src/main/kotlin/kweb/Element.kt

```diff
@@ -299,42 +305,44 @@ open class Element(
                 val element = jsoupDoc.getElementById(this.id)
                 element.appendText(value)
             }
-            canSendInstruction() -> {
-                browser.send(Server2ClientMessage.Instruction(Server2ClientMessage.Instruction.Type.AddText, listOf(id, value)))
+            canSendMessage() -> {
+                val js = "const textNode = document.createTextNode({});document.getElementById({}).appendChild(textNode);"
+                browser.callJsFunction(js, value, id)
             }
             else -> {
-                execute("""
-                {
-                    var ntn=document.createTextNode("${value.escapeEcma()}");
-                    $jsExpression.appendChild(ntn);
-                }
-        """)
+                callJsFunction("""
+                    let id = {}
+                    let value = {}
+                    var ntn=document.createTextNode(value);
+                    document.getElementById(id).appendChild(ntn);
+                """, id, value.escapeEcma())
```

Comment:
Might want to use https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-indent.html here.